components/Materiallist.js

```diff
@@ -0,0 +1,109 @@
+import styled from "styled-components";
+import SwipeToDeleteItem from "./SwipeToDelete";
+import MaterialListForm from "./MaterialListForm";
+
+export default function MaterialList({
+  addNewMaterial,
+  currentEntry,
+  handleMateriallistDelete,
+}) {
+  return (
+    <>
+      <MaterialListForm addNewMaterial={addNewMaterial} />
+      <FlexWrapper>
+        <ListHeader>p</ListHeader>
+        <ListHeader>w</ListHeader>
+        <ListHeader>h</ListHeader>
+        <ListHeader>d</ListHeader>
+        <ListHeader>unit</ListHeader>
+      </FlexWrapper>
+      <List>
+        {currentEntry.dimensions.map((dimension) => (
+          <li key={dimension.id}>
+            <SwipeToDeleteItem
+              handleMateriallistDelete={() =>
+                handleMateriallistDelete(dimension.id)
+              }
+              dimensionId={dimension.id}
+            >
+              <StyledSubList>
+                <Wrapper>
+                  <ListItem>
+                    name: {dimension.name}
+                    <Tooltip>ᗏ Swipe left to delete</Tooltip>
+                  </ListItem>
+                  {dimension.material ? (
+                    <ListItem>material: {dimension.material}</ListItem>
+                  ) : null}
+                </Wrapper>
+                <FlexWrapper>
```

Comment:
Your flex wrapper is a div, not a list

Reply:
styling was difficult, but i actually dont need the wrappers, deleted 👍